docs/integrate/dbeaver/index.md

```diff
@@ -0,0 +1,90 @@
+(dbeaver)=
+# DBeaver
+
+```{div}
+:style: "float: right; margin-left: 0.5em"
+[![](https://upload.wikimedia.org/wikipedia/commons/thumb/b/b5/DBeaver_logo.svg/512px-DBeaver_logo.svg.png){w=120px}](https://dbeaver.io/)
+```
+
+[DBeaver] is a multipurpose cross-platform database tool for developers,
+database administrators, analysts, and everyone working with data.
+
+It is available as an open-source version _DBeaver Community_ and
+as a commercial version _DBeaver PRO_.
+
+
+## Install
+
+::::{grid} 2
+:::{grid-item}
+For connecting to CrateDB, the [CrateDB JDBC Driver] will be used.
+:::
+:::{grid-item}
+![Image](https://github.com/user-attachments/assets/ebd11a53-4d4a-4a9b-bed2-25909e618929){w=480px}
```

Comment:
This may not be up-to-date, there are 2 options legacy and non-legacy
![image](https://github.com/user-attachments/assets/a9d14328-0530-47c9-aaa3-960c731ef484)

Reply:
Thank you. I will probably need to ramp up a DBeaver myself, in order to capture the current state of affairs better. Within this iteration, I just reused existing resources for DBeaver. Maybe I will split the PR into two halves.

Reply:
I've diverted concerns about DBeaver to GH-157.